### DISCUSSION TITLE
TimeStamp/WithLatestForm stuck on 16/17 miliseconds

### DISCUSSION THREAD

**Discussion by Mardornack** [bonsai-rx/bonsai#1300] [Q&A]
Hello again, 
I am trying to print in a "CsWriter Node" XY mouse coordinates sample. I am trying to use "TimeStamp" and "WithLatestFrom" nodes to write the coordinates each 20ms, The thing is, that it is stucked in 16/17ms. It doesn´t matter if I change the time in the nodes, it always marks the same. It works fine with higher time like 1s or 5s for example, but after a certain number it stucks on 16/17ms. Thanks a lot.

**Comment by bruno-f-cruz** [comment]
Can you share the workflow?

**Reply by Mardornack** [reply]
![image](https://user-images.githubusercontent.com/128514957/227953686-4d1bcd64-b1e9-4f2a-9213-ceac1ce7614a.png)
Yes of course! Sorry for the delay, but the code was at my work. 
Thanks a lot!

**Reply by bruno-f-cruz** [reply]
In the future share the .bonsai workflow file. Makes it much easier for people to provide feedback :)

The behavior you are observing is related to the way "Mouse" is sampled in your workflow. Since you are using the input of UpdateFrame to Mouse, it will sample on each UpdateFrame event. I am guessing your screen is probably running at 60Hz, hence the 16-17ms sampling period.
Hope it helps,
Bruno
